src/db/seed.ts

```diff
@@ -33,20 +34,30 @@ const insertUsersSchema = createInsertSchema(schema.users);
 const insertUsersToGroupsSchema = createInsertSchema(schema.usersToGroups);
 
 async function seed(dbPool: NodePgDatabase<typeof schema>) {
-  const randId = randUuid();
+  const randCityFieldId = randUuid();
+  const randAgeFieldId = randUuid();
   const events = await createEvent(dbPool, [
     {
       name: randCity(),
       fields: {
-        [randId]: {
-          id: randId,
-          name: 'submit project',
+        [randCityFieldId]: {
+          id: randCityFieldId,
+          name: 'city',
           type: 'TEXT',
           position: 0,
           validation: {
             required: true,
           },
         },
+        [randAgeFieldId]: {
+          id: randAgeFieldId,
+          name: 'age',
+          type: 'NUMBER',
+          position: 0,
```

Comment:
```suggestion
          position: 1,
```

Reply:
I assume that position will replace field display rank. Therefore, each registration field should have a different position that the frontend will take instead of field display rank to order registration fields.